dev/tests/api-functional/phpunit_rest.xml.dist

```diff
@@ -19,6 +19,10 @@
         <testsuite name="Magento REST web API functional tests">
             <directory suffix="Test.php">testsuite</directory>
             <exclude>testsuite/Magento/GraphQl</exclude>
+            <directory suffix="Test.php">../../../app/code/*/*/Test/Api</directory>
+        </testsuite>
+        <testsuite name="msi">
```

Comment:
Looks like this was only needed to execute tests on `MSI` fork

Reply:
@ishakhsuvarov removed

Reply:
Where is it removed?

Reply:
Done